lib/class-wp-theme-json-gutenberg.php

```diff
@@ -3417,7 +3417,7 @@ protected static function should_override_preset( $theme_json, $path, $override
 
 	/**
 	 * Returns the default slugs for all the presets in an associative array
-	 * whose keys are the preset paths and the leafs is the list of slugs.
+	 * whose keys are the preset paths and the leaves is the list of slugs.
```

Comment:
Note: This typo is already fixed in the core, so a backport changelog should not be necessary.

https://github.com/WordPress/wordpress-develop/blob/07be2445e45066147105206da08acf31bcee974d/src/wp-includes/class-wp-theme-json.php#L3386